client/src/pages/LatexCollection.js

```diff
@@ -141,6 +157,10 @@ class LatexCollection extends Component {
                         columnDefs={this.state.columnDefs}
                         frameworkComponents={this.state.frameworkComponents}
                     ></AgGridReact>
```

Comment:
What is the purpose of frameworkComponents?

Reply:
It was a part of the code in Main Page forgot to remove when copied.

Reply:
Sir this is used to render the payment status.